tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -3064,6 +3064,42 @@ public function userStopRecordingInRoom(string $user, string $identifier, int $s
 		$this->assertStatusCode($this->response, $statusCode);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" store recording file "([^"]*)" in room "([^"]*)" with (\d+)(?: \((v1)\))?$/
+	 */
+	public function userStoreRecordingFileInRoom(string $user, string $file, string $identifier, int $statusCode, string $apiVersion = 'v1'): void {
+		$this->setCurrentUser($user);
+
+		$sipBridgeSharedSecret = 'the secret';
+		$this->setAppConfig('spreed', new TableNode([['sip_bridge_shared_secret', $sipBridgeSharedSecret]]));
+		$validRandom = md5((string) rand());
+		$validChecksum = hash_hmac('sha256', $validRandom . self::$identifierToToken[$identifier], $sipBridgeSharedSecret);
+		$headers = [
+			'TALK_SIPBRIDGE_RANDOM' => $validRandom,
+			'TALK_SIPBRIDGE_CHECKSUM' => $validChecksum,
+		];
+		$options = [
+			'multipart' => [
```

Comment:
@nickvergessen is multipart formdata the best way to do this or we have another best approach, maybe using chunks that could have a best performance with big files?